app/services/model/AutoCompleteItem.scala

```diff
@@ -20,5 +20,5 @@ case class AutoCompleteItem(label: String, value: String)
 
 object AutoCompleteItem {
   def from(countries: List[Country], value: Country => String = _.countryName): List[AutoCompleteItem] =
-    countries map (c => AutoCompleteItem(c.countryName, value(c)))
+    countries map (c => AutoCompleteItem(s"${c.countryName} - ${c.countryCode}", value(c)))
```

Comment:
This Scala syntax sugar is not what we tend to use here.
Fine with me now but let's discuss this